x-pack/plugins/maps/public/routing/routes/list/list.js

```diff
@@ -5,10 +5,14 @@
  */
```

Comment:
if we're going to rename this file, make it consistent with the class name `map_list_view`.